### PR TITLE
Update reward sharing guide in Lisk docs

### LINKED ISSUE #1909: Update reward sharing guide
### Motivation

- The screenshots are outdated, update the screenshots with new features.
- Decrease the commission of a validator to 25 and incentivize validators to choose a lower commission to incentivize staking.

#### Affected branch / URL
`development`

#### Page location
`run-blockchain/reward-sharing.adoc`

### PR DESCRIPTION
**Pull request by chris529**
Closes #1909

### CODE REVIEW

**Review by Incede — APPROVED** [review]

File: docs/modules/ROOT/pages/run-blockchain/reward-sharing.adoc [hunk 1]
```diff
@@ -64,27 +65,27 @@ To share rewards via Lisk Desktop, the validator should be logged in to their ac
...
 . Lisk Desktop will display an *Edit commission successful* message after submitting the request successfully.
 +
-image:run-blockchain/reward-sharing/share-reward-4.png["Edit commission successful",300]
+image:run-blockchain/reward-sharing/share-reward-04.png["Edit commission successful",300]
 . Once the transaction is executed and the block is finalized, the updated commission value will be shown on the *My validator profile*.
 +
-image:run-blockchain/reward-sharing/share-reward-5.png["Updated value",300]
+image:run-blockchain/reward-sharing/share-reward-05.png["Updated value",300]
 
 === Sharing via CLI
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/run-blockchain/reward-sharing.adoc:90]: The **Sharing via CLI** example also needs an update as the value of `commission` is still `85` in the aforementioned example.

File: docs/modules/ROOT/pages/run-blockchain/reward-sharing.adoc [hunk 2]
```diff
@@ -20,6 +20,7 @@ Muhammad Talha <muhammad.talha@lightcurve.io>
 :url_constraints-on-commission-increase: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0070.md#constraints-on-commission-increase
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/run-blockchain/reward-sharing.adoc:20]: Please delete the old images!

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Please consider the following!

File: docs/modules/ROOT/pages/run-blockchain/reward-sharing.adoc (hunk 1 above)
Comment (chris529) [reply docs/modules/ROOT/pages/run-blockchain/reward-sharing.adoc:90]: Good point, now done. 👍

File: docs/modules/ROOT/pages/run-blockchain/reward-sharing.adoc (hunk 2 above)
Comment (chris529) [reply docs/modules/ROOT/pages/run-blockchain/reward-sharing.adoc:20]: Done 👍

File: docs/modules/ROOT/pages/run-blockchain/reward-sharing.adoc [hunk 3]
```diff
@@ -113,7 +114,7 @@ Sidechain Node::
 =====
 +
 . The *validator* should enter their `passphrase` and the `newCommission` value. 
-In this example, the validator is set to keep *85%* of the rewards and share *15%* with their stakers, hence the value of `8500`.
+In this example, the validator is set to keep *75%* of the rewards and share *25%* with their stakers, hence the value of `8500`.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/run-blockchain/reward-sharing.adoc]: Please follow this and change all the values accordingly!
```suggestion
In this example, the validator is set to keep *25%* of the rewards and share *75%* with their stakers, hence the value of `2500`.
```

File: docs/modules/ROOT/pages/run-blockchain/reward-sharing.adoc [hunk 4]
```diff
@@ -68,7 +68,7 @@ To share rewards via Lisk Desktop, the validator should be logged in to their ac
 image:run-blockchain/reward-sharing/share-reward-00.png["Click on edit commission",300]
 +
 . Type `25`, in the *Edit Commission* dialog box, and click on the *Confirm* button.
-In this example, the validator is set to keep *25%* of the rewards as a commission and share the remaining *75%* with their stakers, hence the value of `25`.
+In this example, the validator is set to keep *25%* of the rewards as a commission and share the remaining *75%* with their stakers, hence the value of `2500`.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/run-blockchain/reward-sharing.adoc]:
```suggestion
In this example, the validator is set to keep *25%* of the rewards as a commission and share the remaining *75%* with their stakers, hence the value of `25`.
```

**Review by TalhaMaliktz — APPROVED** [review]
LGTM 👍🏼

**Review by TalhaMaliktz — APPROVED** [review]

**Pull request merged**